aesm-client/src/lib.rs

```diff
@@ -42,23 +37,23 @@ use sgxs::sigstruct::{Attributes, Sigstruct};
 
 include!(concat!(env!("OUT_DIR"), "/mod_aesm_proto.rs"));
 mod error;
-
 use self::aesm_proto::*;
 pub use error::{AesmError, Error, Result};
-
+#[cfg(windows)]
+extern crate winapi;
+#[cfg(windows)]
+extern crate sgx_isa;
```

Comment:
See coding style guide for the order in which to put things at the top of the module